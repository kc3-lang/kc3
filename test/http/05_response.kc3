quote def f = fn () { res = %HTTP.Response{body: "a#{:a}"} }
def f = fn () { res = %HTTP.Response{body: "a#{:a}"} }
