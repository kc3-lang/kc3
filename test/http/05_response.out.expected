require HTTP.Response
true
def f = fn () { res = %HTTP.Response{body: "a#{:a}"} }
f
